api/util/mappers/engine.py

```diff
@@ -419,14 +425,29 @@ def _get_prioritised_feature_states(
 def _get_segment_feature_states(
     segments: Iterable["Segment"],
     environment_id: int,
+    latest_environment_feature_versions: (
+        Iterable["EnvironmentFeatureVersion"] | None
+    ) = None,
```

Comment:
nit: optional argument is not needed here

Reply:
Done.